### ISSUE TITLE
Line 132: *.conf not found

### ISSUE THREAD

**Issue by NikRoCraftYT** [geeks-r-us/anbox-playstore-installer#49]
Terminal Error Code (Ubuntu 20 / 4th June 2020)

./install-playstore.sh: Line 132: /etc/systemd/system/anbox-container-manager.service.d/override.conf: not found

**Comment by thurmancotton** [comment]
I have the same error when running the install script on Debian Buster with Anbox installed via `apt` package.

**Comment by vishalkrv** [comment]
Facing the same issue, can someone please look into it

**Comment by fantostisch** [comment]
It fails because the directory does not exist, you can create it using:
```bash
sudo mkdir /etc/systemd/system/anbox-container-manager.service.d/
```

**Comment by pothos** [comment]
after `mkdir` I also needed to replace the `$SUDO cat >` with `$SUDO tee`

**Comment by LokeshaRamesh** [comment]
Tried and got this

[Service]
ExecStart=
ExecStart=/usr/bin/anbox container-manager --daemon --privileged --data-path=/var/lib/anbox --use-rootfs-overlay
/var/lib/anbox/rootfs-overlay
Overlay no enabled ! Please check error messages!

**Comment by frrahat** [comment]
It worked for me:

1. First run
```
sudo mkdir /etc/systemd/system/anbox-container-manager.service.d/

```

2. Open `install-playstore.sh` in editor
3. Search for the following snippet:
```
if ( [ -d '/var/snap' ] || [ -d '/snap' ] ) && \
	( [ ${ANBOX} = "/snap/bin/anbox" ] || [ ${ANBOX} == /var/lib/snapd/snap/bin/anbox ] );then
```
4. Change it to this (remove the second set of conditions)
```
SNAP_TOP=""
if ( [ -d '/var/snap' ] || [ -d '/snap' ] );then
```

**Comment by pkunzler** [comment]
@fantostisch : 
> sudo mkdir /etc/systemd/system/anbox-container-manager.service.d/
install-playstore.sh succeed after adding write permission to directory:
> sudo chmod a+w /etc/systemd/system/anbox-container-manager.service.d/
Thanks !

**Comment by RiseWith** [comment]
Holy cow!! THANKS! Finally a working solution.

**Comment by Anil-v29** [comment]
> It worked for me:
> 
>     1. First run
> 
> 
> ```
> sudo mkdir /etc/systemd/system/anbox-container-manager.service.d/
> ```
> 
>     1. Open `install-playstore.sh` in editor
> 
>     2. Search for the following snippet:
> 
> 
> ```
> if ( [ -d '/var/snap' ] || [ -d '/snap' ] ) && \
> 	( [ ${ANBOX} = "/snap/bin/anbox" ] || [ ${ANBOX} == /var/lib/snapd/snap/bin/anbox ] );then
> ```
> 
>     1. Change it to this (remove the second set of conditions)
> 
> 
> ```
> if ( [ -d '/var/snap' ] || [ -d '/snap' ] );then
> ```

Thanks! It works

**Comment by THEENGINEERNT** [comment]
its not working

**Comment by geeks-r-us** [comment]
Should be fixed with #70

**Comment by rasindunimsara6** [comment]
> It worked for me:
> 
> 1. First run
> 
> ```
> sudo mkdir /etc/systemd/system/anbox-container-manager.service.d/
> ```
> 
> 2. Open `install-playstore.sh` in editor
> 3. Search for the following snippet:
> 
> ```
> if ( [ -d '/var/snap' ] || [ -d '/snap' ] ) && \
> 	( [ ${ANBOX} = "/snap/bin/anbox" ] || [ ${ANBOX} == /var/lib/snapd/snap/bin/anbox ] );then
> ```
> 
> 4. Change it to this (remove the second set of conditions)
> 
> ```
> if ( [ -d '/var/snap' ] || [ -d '/snap' ] );then
> ```

this actually worked for me too. thank you very much
